### DISCUSSION TITLE
Invalid login attempt.

### DISCUSSION THREAD

**Discussion by ghost** [immense/Remotely#679] [General]
how can i solve this problem
<img width="724" alt="Screenshot 2023-07-06 at 5 00 46 PM" src="https://github.com/immense/Remotely/assets/56769514/2c24c759-3afc-4d7d-b045-3103f0cdc3fa">

**Comment by roeberp** [comment]
I am getting the same error. Some details:

I had this container turned off for several months and Watchtower hadn't been updating it. Last version I had was from sometime in February. Well, I had a need for Remotely so in preparation, I turned it back on and was able to login yesterday. It looks like Watchtower updated it last night and now I can no longer log in.

Any ideas on how to fix this? Any help is much appreciated!

**Reply by roeberp** [reply]
It turns out that my config was wiped, presumably during the upgrade.  I had to re-register and it seems fine now.
